From 99cdb649cb2c10bec86760db7b59731e77b10b88 Mon Sep 17 00:00:00 2001
From: Kaushlendra Kumar <kaushlendra.kumar@intel.com>
Date: Mon, 8 Aug 2022 11:07:31 +0530
Subject: [PATCH] Power Optimization

1. remove un-used applications from system
2. disable home screen search

Tracked-On: OAM-103484
Signed-off-by: Kaushlendra Kumar <kaushlendra.kumar@intel.com>
---
 target/product/handheld_product.mk    |  7 -------
 target/product/handheld_system.mk     | 15 ---------------
 target/product/handheld_system_ext.mk |  1 -
 3 files changed, 23 deletions(-)

diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index 2199c570a0..c10bfe2e4f 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -22,17 +22,10 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/media_product.mk)
 
 # /product packages
 PRODUCT_PACKAGES += \
-    Browser2 \
-    Calendar \
-    Camera2 \
-    Contacts \
-    DeskClock \
-    Gallery2 \
     LatinIME \
     Music \
     OneTimeInitializer \
     preinstalled-packages-platform-handheld-product.xml \
-    QuickSearchBox \
     SettingsIntelligence \
     frameworks-base-overlays
 
diff --git a/target/product/handheld_system.mk b/target/product/handheld_system.mk
index b7a2d0dad0..e6068b63c3 100644
--- a/target/product/handheld_system.mk
+++ b/target/product/handheld_system.mk
@@ -32,46 +32,31 @@ $(call inherit-product-if-exists, frameworks/base/data/keyboards/keyboards.mk)
 $(call inherit-product-if-exists, frameworks/webview/chromium/chromium.mk)
 
 PRODUCT_PACKAGES += \
-    BasicDreams \
-    BlockedNumberProvider \
     Bluetooth \
     BluetoothMidiService \
-    BookmarkProvider \
-    BuiltInPrintService \
-    CalendarProvider \
     cameraserver \
     CameraExtensionsProxy \
-    CaptivePortalLogin \
     CertInstaller \
     clatd \
     DocumentsUI \
     DownloadProviderUi \
-    EasterEgg \
     ExternalStorageProvider \
     FusedLocation \
     InputDevices \
     KeyChain \
     librs_jni \
     ManagedProvisioning \
-    MmsService \
     MtpService \
     MusicFX \
-    NfcNci \
     PacProcessor \
     PrintRecommendationService \
     PrintSpooler \
     ProxyHandler \
     screenrecord \
-    SecureElement \
     SharedStorageBackup \
-    SimAppDialog \
-    Telecom \
-    TelephonyProvider \
-    TeleService \
     Traceur \
     UserDictionaryProvider \
     VpnDialogs \
-    vr \
 
 
 PRODUCT_SYSTEM_SERVER_APPS += \
diff --git a/target/product/handheld_system_ext.mk b/target/product/handheld_system_ext.mk
index d935fbfddf..8406fee939 100644
--- a/target/product/handheld_system_ext.mk
+++ b/target/product/handheld_system_ext.mk
@@ -27,4 +27,3 @@ PRODUCT_PACKAGES += \
     Settings \
     StorageManager \
     SystemUI \
-    WallpaperCropper \
-- 
2.33.0

